week-3/Homework/mandatory/1-oxygen-levels.js

```diff
@@ -9,7 +9,12 @@ To be safe, they need to land on the first unamed planet that has Oxygen levels
 Write a function that finds the oxygen level of the first safe planet - Oxygen between 19.5% and 23.5%
 */
 
-function safeLevels() {
+
+
+
+function safeLevels(levels) {
+return   levels.sort().find( level => parseFloat(level) > 19.5);
```

Comment:
This is an interesting solution. I'm not 100% sure it's correct though. If we add a test with this test data then it would pick the 4th planet not the 3rd planet:

```js
["24.2%", "11.3%", "23.1%", "19.9%", "29.3%", "20.2%"]
```

Additionally, using a sort is also likely to be much less performant than a simple loop. The sort will have to go through all of the data at least once. With a loop, or with one of the array methods, in the worst case you would go through all of the data once (and once only). You could improve this further by returning early once you find a match.

(although with this data set it